homeassistant/components/kdeconnect/binary_sensor.py

```diff
@@ -0,0 +1,147 @@
+"""Support for KDE Connect binary sensors."""
+from typing import cast
+
+from pykdeconnect.client import KdeConnectClient
+from pykdeconnect.devices import KdeConnectDevice
+from pykdeconnect.plugin_registry import PluginRegistry
+from pykdeconnect.plugins.battery import BatteryReceiverPlugin
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import STATE_ON
+from homeassistant.core import HomeAssistant, State
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import KdeConnectEntity, KdeConnectPluginEntity
+from .const import DATA_KEY_CLIENT, DATA_KEY_DEVICES, DOMAIN
+
+BINARY_SENSOR_TYPES = [
+    BinarySensorEntityDescription(
+        key="battery_charging",
+        name="Battery Charging",
+        device_class=BinarySensorDeviceClass.BATTERY_CHARGING,
+    ),
+    BinarySensorEntityDescription(
+        key="battery_low",
+        name="Battery Low",
+        device_class=BinarySensorDeviceClass.BATTERY,
+    ),
+]
+
+BINARY_SENSOR_PLUGINS = {
+    "battery_charging": BatteryReceiverPlugin,
+    "battery_low": BatteryReceiverPlugin,
+}
+
```

Comment:
Do you need a separate dict for these? Or could you add them to the `BinarySensorEntityDescription`? (you can extend this class, see Tuya integration for an example)

Reply:
Done.